packages/cli/src/controller/deploy-controller.ts

```diff
@@ -185,11 +194,163 @@ export async function imageVersions(name: string, version: string, authToken: st
     errorHandle(e, 'Failed to get image:');
   }
 }
+
 export function splitEndpoints(endpointStr: string): string[] {
   return endpointStr.split(',').map((e) => e.trim());
 }
+
 export interface EndpointType {
   network: string;
   endpoint: string;
   chainId: string;
 }
+
+export function splitMultichainDataFields(fieldStr: string): MultichainDataFieldType {
+  let result: MultichainDataFieldType = {};
+
+  splitEndpoints(String(fieldStr)).forEach(unparsedRow => {
+    let regexpResult: string[] = unparsedRow.match(/(.*?):(.*)/);
+    if (regexpResult) {
+      regexpResult = Object.values(regexpResult);
+      if (regexpResult && regexpResult.length == 6 && ['http','https','ws','wss'].indexOf(regexpResult[1])===-1 ) {
+        result[regexpResult[1]] = regexpResult[2];
+      }
+    }
+  });
+
+  return result;
+}
+
+export function defaultCommandFlags(): & FlagInput<{ [flag: string]: any; }>{
```

Comment:
This can be a constant rather than a function and the type can be improved, this will tie the interface to the flags

```ts
export const DefaultDeployFlags: FlagInput<DeploymentFlagsInterface> = {
 //...Flags
}

```